app/src/androidTest/java/com/github/se/eventradar/qrCode/qrCodeScanTicketUiTest.kt

```diff
@@ -91,6 +90,117 @@ class QrCodeScanTicketUiTest : TestCase(kaspressoBuilder = Kaspresso.Builder.wit
     mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
   }
 
+  @Test
+  fun screenDisplaysNavigationElementsCorrectly() = run {
+    every { mockViewModel.uiState } returns myEventUiState
+    onComposeScreen<QrCodeScanTicketUiScreen>(composeTestRule) {
+      composeTestRule.setContent { QrCodeTicketUi(mockViewModel, mockNavActions) }
+      bottomNavMenu.assertIsDisplayed()
+      goBackButton.assertIsDisplayed()
+    }
+  }
```

Comment:
I think the tab button for "My Event" and "Scan Ticket" should be added as Navigation elements and be tested here too